building-with-ocean/build-a-marketplace/forking-ocean-market.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Forking Ocean Market
+order: 1
+hideLanguageSelector: true
+description: Forking and running Ocean Market locally.
+featuredImage: images/creatures/mantaray/mantaray-full@2x.png
+---
+
+# Forking and running Ocean Market locally
+
+Have you ever thought about monetizing the data that your organization generates? You might imagine that this involves dealing with data brokers, but with Ocean you can sell your data directly to your customers with no third party in-between. Ocean makes this all super easy for you with some pretty cool tech under the hood.
```

Comment:
Initially, I thought that I didn't change the page cause the main README starts with a sentence that is similar to this one: 'Have you ever thought about monetizing digital assets ov...'. Maybe we can change a bit the wording here. Just a suggestion

Reply:
Done